cfpb/consumerfinance.gov
# HOLD: Deprecate Paying for College feedback form

This commit deprecates the standalone feedback form for Paying for College / Understanding your financial aid offer. This page currently lives at

https://www.consumerfinance.gov/paying-for-college2/understanding-your-financial-aid-offer/feedback/

and is linked at the bottom of the offer review page.

This commit retires that page and removes the link to it, plus removes all related frontend assets.

## How to test this PR

To test with an empty database, add these test fixtures:

- `cfgov/manage.py loaddata cfgov/paying_for_college/fixtures/test_program.json`
- `cfgov/manage.py loaddata cfgov/paying_for_college/fixtures/test_constants.json`

Then visit this URL with a local server:

http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=408039&oid=0000000000000000000000000000000000000000&pid=981

Click through the various prompts and see that the feedback link on the bottom of the page is now gone.

## Screenshots

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/654645/163862793-4058e3f5-1d35-49b5-90c5-5016bd10f3e2.png)|![image](https://user-images.githubusercontent.com/654645/163862590-f868f680-4dec-4860-9672-9eebff51c3c7.png)|

## Notes and todos

This does not remove the `paying_for_college.Feedback` model; that will be cleaned up in the future.

## Checklist
